### PR TITLE
Minutes 2023-12-06

### PR DESCRIPTION
**Pull request by elf-pavlik**

### CODE REVIEW

File: meetings/2023-12-06.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+### Solid CG Chair Election Procedure
+URL: https://github.com/solid/specification/discussions/582
+
+* eP: One more week to vote: https://www.w3.org/2002/09/wbs/110151/cfb1563d-40eb-4c65-9859-30ee63e739bb/
+* eP: Pleaes double check if your name appears on https://github.com/w3c-cg/solid/blob/main/elections/2023/eligible-voters.txt
+    * If your name is not listed, please raise an issue or make a PR
+* eP: Any CG member can use the online poll, only votes cast by names listed above will be counted! We plan to provide finalized list to W3C Monday-ish, last chance to raise issue of your name is not on it and you believe it should be.
+* AC: each organization has one vote, for example for Inrupt I am the designated voter. You can see Hadrian is not on the list of eligible voters.
+* ML: to be clear, if I am in an organization, I cannot vote?
```
Comment (csarven) [inline meetings/2023-12-06.md:111]: @lecoqlibre @andreiciortea , to make sure there is no oversight, e.g., your affiliations not being up to date, may I ask you to both confirm that it is up to date in https://www.w3.org/groups/cg/solid/participants/ , i.e., you're with "Institut National de Recherche en Informatique et en Automatique (Inria)".

If so, great, but also note that as it stands, only @andreiciortea can vote ( as listed inhttps://github.com/w3c-cg/solid/blob/main/elections/2023/eligible-voters.txt ) based on the information we've gathered in https://github.com/w3c-cg/solid/blob/main/elections/2023/affiliation-designated-voters.md and asked the CG to verify in https://lists.w3.org/Archives/Public/public-solid/2023Nov/0075.html

If either of your affiliations should be different, please update that, and let us know about it in an issue at https://github.com/w3c-cg/solid/ . If wish to change the current designated voter from @andreiciortea to @lecoqlibre , you can let us know as well, and perhaps create a PR against https://github.com/w3c-cg/solid/blob/main/elections/2023/affiliation-designated-voters.md .

Comment (FabienGandon) [reply meetings/2023-12-06.md:111]: @csarven as Inria AC Rep I confirm the affiliation of @andreiciortea to Inria (and his inria email is used for W3C account) and that he is voting for Inria

Comment (csarven) [reply meetings/2023-12-06.md:111]: Great, thanks for confirming Andrei and Fabien. I'm glad this resolves the potential concern. So... happy voting =)

File: meetings/2023-12-06.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
+# W3C Solid Community Group: Weekly
+
+* Date: 2023-12-06T14:00:00Z
+* Call: https://meet.jit.si/solid-cg
+* Chat: https://matrix.to/#/#solid_specification:gitter.im
+* Repository: https://github.com/solid/specification
+* Status: Draft
```
Comment (csarven) [inline meetings/2023-12-06.md:7]:
```suggestion
* Status: Published
```

**Pull request merged**